yellowbrick/features/importances.py

```diff
@@ -168,7 +171,12 @@ def draw(self, **kwargs):
         pos = np.arange(self.features_.shape[0]) + 0.5
 
         # Plot the bar chart
-        self.ax.barh(pos, self.feature_importances_, align='center')
+        #raise Exception(self.feature_importances_)
+        # Plot the bar chart
+
+
+        colors = np.array([self.color_pos if v > 0 else self.color_neg for v in self.feature_importances_])
```

Comment:
Could this line be changed to reflect that we consider Zero to be a positive number and should be colored as such.  

`if v >= 0`